model/feature_flags.go

```diff
@@ -104,6 +107,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.SendWelcomePost = true
 	f.PostPriority = false
 	f.PeopleProduct = false
+	f.ReduceOnBoardingTaskList = true
```

Comment:
If `true` enables new functionality, I think the default should be `false` (ie, use old functionality)